fe/java-udf/src/main/java/org/apache/doris/jni/PaimonJniScanner.java

```diff
@@ -0,0 +1,187 @@
+import org.apache.paimon.hive.mapred.PaimonInputSplit;
+import org.apache.paimon.options.CatalogOptions;
+import org.apache.paimon.options.ConfigOption;
+import org.apache.paimon.options.ConfigOptions;
+import org.apache.paimon.options.Options;
+import org.apache.paimon.reader.RecordReader;
+import org.apache.paimon.table.Table;
+import org.apache.paimon.table.source.ReadBuilder;
+import org.apache.paimon.table.source.TableRead;
+
+import java.io.ByteArrayInputStream;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+
+
+public class PaimonJniScanner extends JniScanner {
+    private static final Logger LOG = Logger.getLogger(PaimonJniScanner.class);
+
+    private final String metastoreUris;
+    private final String warehouse;
+    private final String dbName;
+    private final String tblName;
+    private final String[] ids;
+    private final long splitAddress;
+    private final int lengthByte;
+    private PaimonInputSplit paimonInputSplit;
+    private Table table;
+    private RecordReader<InternalRow> reader;
+    private final PaimonColumnValue columnValue = new PaimonColumnValue();
+
+    public PaimonJniScanner(int batchSize, Map<String, String> params) {
+        metastoreUris = params.get("hive.metastore.uris");
+        warehouse = params.get("warehouse");
+        splitAddress = Long.parseLong(params.get("split_byte"));
+        lengthByte = Integer.parseInt(params.get("length_byte"));
+        LOG.info("splitAddress:" + splitAddress);
+        LOG.info("lengthByte:" + lengthByte);
+        dbName = params.get("db_name");
+        tblName = params.get("table_name");
+        String[] requiredFields = params.get("required_fields").split(",");
+        String[] types = params.get("columns_types").split(",");
+        ids = params.get("columns_id").split(",");
+        ColumnType[] columnTypes = new ColumnType[types.length];
+        for (int i = 0; i < types.length; i++) {
+            columnTypes[i] = ColumnType.parseType(requiredFields[i], types[i]);
+        }
+        ScanPredicate[] predicates = new ScanPredicate[0];
+        if (params.containsKey("push_down_predicates")) {
+            long predicatesAddress = Long.parseLong(params.get("push_down_predicates"));
+            if (predicatesAddress != 0) {
+                predicates = ScanPredicate.parseScanPredicates(predicatesAddress, columnTypes);
+                LOG.info("MockJniScanner gets pushed-down predicates:  " + ScanPredicate.dump(predicates));
+            }
+        }
+        initTableInfo(columnTypes, requiredFields, predicates, batchSize);
+    }
+
+    @Override
+    public void open() throws IOException {
+        getCatalog();
+        // 拿 []byte 反序列化成 split
+        byte[] splitByte = new byte[lengthByte];
+        OffHeap.copyMemory(null, splitAddress, splitByte, OffHeap.BYTE_ARRAY_OFFSET, lengthByte);
+        LOG.info("splitBytes:" + Arrays.toString(splitByte));
```

Comment:
```suggestion
        LOG.debug("splitBytes: {}", Arrays.toString(splitByte));
```

And if `Arrays.toString(splitByte)` is expensive, better if `if LOG.isDebugEnable()`